jenkins-master/jobs/ondemand_update/config.xml

```diff
@@ -61,55 +50,38 @@
-            </a>
-          </choices>
-        </hudson.model.ChoiceParameterDefinition>
         <hudson.model.StringParameterDefinition>
-          <name>OVERRIDE_UPDATE_URL</name>
-          <description>The URL used by Firefox when checking for updates.</description>
+          <name>REVISION</name>
+          <description>The expected revision of Firefox after the upgrade, which is also used for the report to Treeherder.</description>
           <defaultValue>None</defaultValue>
         </hudson.model.StringParameterDefinition>
-        <hudson.model.ChoiceParameterDefinition>
-          <name>PLATFORM</name>
-          <description>The platform to run the test on.</description>
-          <choices class="java.util.Arrays$ArrayList">
-            <a class="string-array">
-              <string>linux</string>
-              <string>linux64</string>
-              <string>mac</string>
-              <string>mac64</string>
-              <string>win32</string>
-              <string>win64</string>
-            </a>
-          </choices>
-        </hudson.model.ChoiceParameterDefinition>
-        <hudson.model.StringParameterDefinition>
-          <name>REPORT_URL</name>
-          <description>URL of the Mozmill Dashboard to send reports to.</description>
-          <defaultValue></defaultValue>
-        </hudson.model.StringParameterDefinition>
         <hudson.model.StringParameterDefinition>
           <name>TARGET_BUILD_ID</name>
           <description>The expected build id of Firefox after the update.</description>
-          <defaultValue></defaultValue>
+          <defaultValue>None</defaultValue>
         </hudson.model.StringParameterDefinition>
         <hudson.model.StringParameterDefinition>
-          <name>VERSION</name>
-          <description>The version of Firefox.</description>
-          <defaultValue></defaultValue>
+          <name>TEST_PACKAGES_URL</name>
+          <description>The URL of the test_packages.json file for the given build.</description>
+          <defaultValue>None</defaultValue>
         </hudson.model.StringParameterDefinition>
       </parameterDefinitions>
     </hudson.model.ParametersDefinitionProperty>
+    <EnvInjectJobProperty plugin="envinject@1.88">
+      <info>
+        <propertiesFilePath>$JENKINS_HOME/../.jenkins.properties</propertiesFilePath>
+        <propertiesContent>NSPR_LOG_MODULES=timestamp,nsHttp:3,nsHostResolver:5,append
+NSPR_LOG_FILE=http.log</propertiesContent>
+        <loadFilesFromMaster>true</loadFilesFromMaster>
+      </info>
+      <on>true</on>
+      <keepJenkinsSystemVariables>true</keepJenkinsSystemVariables>
+      <keepBuildVariables>true</keepBuildVariables>
+      <contributors/>
+    </EnvInjectJobProperty>
   </properties>
   <scm class="hudson.plugins.cloneworkspace.CloneWorkspaceSCM" plugin="clone-workspace-scm@0.5">
-    <parentJobName>get_mozmill-tests</parentJobName>
+    <parentJobName>get_mozharness</parentJobName>
```

Comment:
As said, this PR already landed last Friday given that it was blocking QA signoff for the Firefox 46.0b1 release. This is a post-review. With my landed mozharness removal patch, this is already gone on master now.